src/components/ui-widgets/GraphPill.tsx

```diff
@@ -139,7 +139,7 @@ export default function GraphPill(props: GraphPillProps) {
   `;
 
   return (
-    <li css={graphPillCss} {...rest}>
```

Comment:
This is probably worthwhile to keep around if we want to add styling directly to the `<li>` element. Also deleting this violates the `React.LiHTMLAttributes<HTMLLIElement>` type.

Reply:
Thanks for the comment! I removed it to remove the warning. But, I think there is a better way.